3rd-party-licenses/assets-licenses.txt

```diff
@@ -489,3 +489,92 @@ public licenses.
 
 Creative Commons may be contacted at creativecommons.org.
 
+--------------------------------------------------------------------------------
+polyDisplay (polyDisplay-Semi1.0) and polyDisplay Sharp (polyDisplay-SemiSharp1.0)
```

Comment:
Let me nitpick a little tiny bit here... It's probably convenient if you use the exact filename here. If this is the name that's contained in the file metadata, you might want to refer to both.

Reply:
Changed! let me know if it's correct now

Reply:
Perfect! Thanks!